backtrader/writer.py

```diff
@@ -105,7 +105,7 @@ def __init__(self):
         self.headers = list()
         self.values = list()
 
-    def start(self):
+    def _start(self):
```

Comment:
Maybe rename to something like `_start_output` (or something) to better differentiate from the regular function `start`?